README.md

```diff
@@ -566,15 +566,3 @@ point that out explicitly and clearly in the associated patches and Cc
   An initial group internal RFC exists in
   (https://github.com/quitschbo/linux/tree/devcg_guard_rfc).
   See commit message for more implementation specific details.
-
-* In a similar vein to `AT_FDCWD`, add `AT_FDROOT` which causes relative and 
-  absolute paths to get interpreted relative to "/". 
```

Comment:
i would find this extremely useful. kinda sucks that you always have to open a root fd first, before you can resolve relative paths relative to root

Reply:
We have an api that allows you to do that with openat2() with detailed control over lookup behavior. Introducing would introduce yet another potentially incompatible variant of similar behavior. It also has regressions risks due to it having to be an arbitrary negative number and is in general hacky.

Reply:
i don't see how you could use openat2() like that, unless you explicitly open an fd for the root fs first. and thinks like unlinkat(), renameat(), mknodat(), symlinkat() and so on all aren't covered by openat2() anyway.

Reply:
For most syscalls openat2() will be used to lookup the target file descriptor and then you call whatever *at variant on the file descriptor. Opening the rootfs file descriptor is especially not a sufficient argument to retroactively plumb AT_FDROOT with similar semantics into legacy system calls. That's just not going to happen. FWIW, even the adition of new O_* flags was rejected and why we ended up with RESOLVE_* flags. IOW, the future is openat2() feel free to propose new features legacy openat calls are unlikely to receive them. Sorry.